app/src/main/java/ru/vssemikoz/newsfeed/adapters/NewsFeedAdapter.java

```diff
@@ -0,0 +1,55 @@
+package ru.vssemikoz.newsfeed.adapters;
+
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+import androidx.annotation.NonNull;
+import androidx.recyclerview.widget.RecyclerView;
+
+import java.util.List;
+
+import ru.vssemikoz.newsfeed.R;
+import ru.vssemikoz.newsfeed.models.NewsItem;
+
+public class NewsFeedAdapter extends RecyclerView.Adapter<NewsFeedAdapter.NewsViewHolder> {
+
+    private List<NewsItem> newsList;
+
+    public NewsFeedAdapter(List<NewsItem> newsList){
+        this.newsList = newsList;
+    }
+    @NonNull
+    @Override
+    public NewsFeedAdapter.NewsViewHolder onCreateViewHolder(@NonNull ViewGroup parent, int viewType) {
+        LayoutInflater inflater = LayoutInflater.from(parent.getContext());
```

Comment:
Let's combine this and next line and remove redundant variables. We don't need a LayoutInflater as a variable here.